src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -58,4 +63,14 @@ public FindCommand parse(String args) throws ParseException {
                 new DescriptionContainsKeywordsPredicate(descriptionKeywords));
     }
 
+    private void checkForNulls(List<String>... lists) throws ParseException {
+        for (List<String> list : lists) {
+            for (String str: list) {
+                if (str.trim().isEmpty()) {
+                    throw new ParseException(
+                            String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+                }
```

Comment:
code quality use streams instead, map, filter and then anyMatch. Then an if statemen to throw an errror